examples/website/collision-filter/app.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import {createRoot} from 'react-dom/client';
+import {Map} from 'react-map-gl';
+import maplibregl from 'maplibre-gl';
+import DeckGL from '@deck.gl/react';
+import {GeoJsonLayer, TextLayer} from '@deck.gl/layers';
+import roads from './data/ne_10_roads_filtered_usa_california.json';
```

Comment:
Why don't we use a bigger dataset @cayetanobv? An example with only 20 roads is not very impressive. We can use a bigger area or use another country, like Spain for instance.

Reply:
Finally, Mexico is the winner. It works perfectly.